src/apps/acb2wavs/acb2wavs.cpp

```diff
@@ -174,6 +180,11 @@ int ProcessAllBinaries(uint32_t formatVersion, const HCA_DECODER_CONFIG &dc, con
     for (auto &entry : archive->GetFiles()) {
         auto &record = entry.second;
         auto extractFileName = CAcbFile::GetSymbolicFileNameFromCueId(record.cueId);
```

Comment:
Code structure. Multiple assignings are not necessary.

```cpp
string s;

if (...) {
    s = ...
} else {
    s = ...
}
```

`std::string` will handle the by-value returns of `GetSymbolicFileNameFromCueId` and `GetCueNameFromCueId`.
Also in this case, do not call `ReplaceExtension` twice (though it does not panic). Beware of semantics.